src/main/java/com/contrastsecurity/http/TraceFilterForm.java

```diff
@@ -92,6 +92,8 @@ public String toString() {
   private int limit;
   private int offset;
   private String sort;
+  private Boolean tracked;
+  private Boolean untracked;
```

Comment:
Can we add some comments indicating what tracked and untracked stand for? Just so future code readers are not confused.